src/plugins/usage_collection/server/usage_counters/rollups/rollups.ts

```diff
@@ -45,8 +49,8 @@ export async function rollUsageCountersIndices(
     return await Promise.all(
       docsToDelete.map(({ id, type, namespaces }) =>
         namespaces?.[0]
-          ? savedObjectsClient.delete(type, id, { namespace: namespaces[0] })
-          : savedObjectsClient.delete(type, id)
+          ? internalRepository.delete(type, id, { namespace: namespaces[0] })
+          : internalRepository.delete(type, id)
```

Comment:
nit: we can do it in a follow up: but we can now use the bulkDelete API :)

Reply:
We'll see about that, cause I believe the `bulkDelete` does not allow deleting from multiple `namespaces`.

Reply:
It will when you add 'force'. Please check that the option hasn't changed.